### ISSUE TITLE
Missing device configuration: Eurotronics Temperature & Humidity Sensor Z-Wave Plus

### ISSUE THREAD

**Issue by amrheing** [AlCalzone/node-zwave-js#3468]
### Checklist

- [X] It is not in the [configuration DB](https://devices.zwave-js.io/?search=3)

- [X] It was not [merged recently](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Apr+is%3Aclosed) or has a [pending PR](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Aopen+is%3Apr)

### Which device is missing?

Eurotronics Temperature & Humidity Sensor Z-Wave Plus

### Manufacturer ID

0x0148

### Product Type

0x0004

### Product ID

0x0001

### Firmware Version

0.1

### Do you have a manual?

[TFS-Z-Wave-Plus_BDA_web_EN.pdf](https://github.com/zwave-js/node-zwave-js/files/7278288/TFS-Z-Wave-Plus_BDA_web_EN.pdf)

Debug Info zwavejs2mqtt controller:

```
{
  "id": 52,
  "name": "",
  "loc": "",
  "values": [
    {
      "id": "52-49-0-Humidity",
      "nodeId": 52,
      "commandClass": 49,
      "commandClassName": "Multilevel Sensor",
      "endpoint": 0,
      "property": "Humidity",
      "propertyName": "Humidity",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Humidity",
      "ccSpecific": {
        "sensorType": 5,
        "scale": 0
      },
      "stateless": false,
      "unit": "%",
      "list": false,
      "value": 50,
      "lastUpdate": 1633352725582,
      "newValue": 50
    },
    {
      "id": "52-49-0-Air temperature",
      "nodeId": 52,
      "commandClass": 49,
      "commandClassName": "Multilevel Sensor",
      "endpoint": 0,
      "property": "Air temperature",
      "propertyName": "Air temperature",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Air temperature",
      "ccSpecific": {
        "sensorType": 1,
        "scale": 0
      },
      "stateless": false,
      "unit": "°C",
      "list": false,
      "value": 22.5,
      "lastUpdate": 1633353451861,
      "newValue": 22.5
    },
    {
      "id": "52-49-0-Dew point",
      "nodeId": 52,
      "commandClass": 49,
      "commandClassName": "Multilevel Sensor",
      "endpoint": 0,
      "property": "Dew point",
      "propertyName": "Dew point",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Dew point",
      "ccSpecific": {
        "sensorType": 11,
        "scale": 0
      },
      "stateless": false,
      "unit": "°C",
      "list": false,
      "value": 12.5,
      "lastUpdate": 1633352389760,
      "newValue": 12.5
    },
    {
      "id": "52-128-0-level",
      "nodeId": 52,
      "commandClass": 128,
      "commandClassName": "Battery",
      "endpoint": 0,
      "property": "level",
      "propertyName": "level",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Battery level",
      "stateless": false,
      "min": 0,
      "max": 100,
      "unit": "%",
      "list": false,
      "value": 100,
      "newValue": 100
    },
    {
      "id": "52-128-0-isLow",
      "nodeId": 52,
      "commandClass": 128,
      "commandClassName": "Battery",
      "endpoint": 0,
      "property": "isLow",
      "propertyName": "isLow",
      "type": "boolean",
      "readable": true,
      "writeable": false,
      "label": "Low battery level",
      "stateless": false,
      "list": false,
      "value": false,
      "newValue": false
    },
    {
      "id": "52-132-0-wakeUpInterval",
      "nodeId": 52,
      "commandClass": 132,
      "commandClassName": "Wake Up",
      "endpoint": 0,
      "property": "wakeUpInterval",
      "propertyName": "wakeUpInterval",
      "type": "number",
      "readable": false,
      "writeable": true,
      "label": "Wake Up interval",
      "default": 3600,
      "stateless": false,
      "min": 600,
      "max": 86400,
      "step": 120,
      "list": false,
      "value": 3600,
      "newValue": 3600
    },
    {
      "id": "52-132-0-controllerNodeId",
      "nodeId": 52,
      "commandClass": 132,
      "commandClassName": "Wake Up",
      "endpoint": 0,
      "property": "controllerNodeId",
      "propertyName": "controllerNodeId",
      "type": "any",
      "readable": true,
      "writeable": false,
      "label": "Node ID of the controller",
      "stateless": false,
      "list": false,
      "value": 1,
      "lastUpdate": 1633337998719,
      "newValue": 1
    },
    {
      "id": "52-134-0-libraryType",
      "nodeId": 52,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "libraryType",
      "propertyName": "libraryType",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Library type",
      "stateless": false,
      "list": true,
      "states": [
        {
          "text": "Unknown",
          "value": 0
        },
        {
          "text": "Static Controller",
          "value": 1
        },
        {
          "text": "Controller",
          "value": 2
        },
        {
          "text": "Enhanced Slave",
          "value": 3
        },
        {
          "text": "Slave",
          "value": 4
        },
        {
          "text": "Installer",
          "value": 5
        },
        {
          "text": "Routing Slave",
          "value": 6
        },
        {
          "text": "Bridge Controller",
          "value": 7
        },
        {
          "text": "Device under Test",
          "value": 8
        },
        {
          "text": "N/A",
          "value": 9
        },
        {
          "text": "AV Remote",
          "value": 10
        },
        {
          "text": "AV Device",
          "value": 11
        }
      ],
      "value": 3,
      "newValue": 3
    },
    {
      "id": "52-134-0-protocolVersion",
      "nodeId": 52,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "protocolVersion",
      "propertyName": "protocolVersion",
      "type": "string",
      "readable": true,
      "writeable": false,
      "label": "Z-Wave protocol version",
      "stateless": false,
      "list": false,
      "value": "5.3",
      "newValue": "5.3"
    },
    {
      "id": "52-134-0-firmwareVersions",
      "nodeId": 52,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "firmwareVersions",
      "propertyName": "firmwareVersions",
      "type": "string[]",
      "readable": true,
      "writeable": false,
      "label": "Z-Wave chip firmware versions",
      "stateless": false,
      "list": false,
      "value": [
        "0.1"
      ],
      "lastUpdate": 1633337998722,
      "newValue": [
        "0.1"
      ]
    },
    {
      "id": "52-134-0-hardwareVersion",
      "nodeId": 52,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "hardwareVersion",
      "propertyName": "hardwareVersion",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Z-Wave chip hardware version",
      "stateless": false,
      "list": false,
      "value": 1,
      "newValue": 1
    },
    {
      "id": "52-114-0-manufacturerId",
      "nodeId": 52,
      "commandClass": 114,
      "commandClassName": "Manufacturer Specific",
      "endpoint": 0,
      "property": "manufacturerId",
      "propertyName": "manufacturerId",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Manufacturer ID",
      "stateless": false,
      "min": 0,
      "max": 65535,
      "list": false,
      "value": 328,
      "newValue": 328
    },
    {
      "id": "52-114-0-productType",
      "nodeId": 52,
      "commandClass": 114,
      "commandClassName": "Manufacturer Specific",
      "endpoint": 0,
      "property": "productType",
      "propertyName": "productType",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Product type",
      "stateless": false,
      "min": 0,
      "max": 65535,
      "list": false,
      "value": 4,
      "newValue": 4
    },
    {
      "id": "52-114-0-productId",
      "nodeId": 52,
      "commandClass": 114,
      "commandClassName": "Manufacturer Specific",
      "endpoint": 0,
      "property": "productId",
      "propertyName": "productId",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Product ID",
      "stateless": false,
      "min": 0,
      "max": 65535,
      "list": false,
      "value": 1,
      "newValue": 1
    }
  ],
  "groups": [
    {
      "text": "Lifeline",
      "endpoint": 0,
      "value": 1,
      "maxNodes": 5,
      "isLifeline": true,
      "multiChannel": false
    },
    {
      "text": "Temperature Report",
      "endpoint": 0,
      "value": 2,
      "maxNodes": 5,
      "isLifeline": false,
      "multiChannel": false
    }
  ],
  "neighbors": [],
  "ready": true,
  "available": true,
  "hassDevices": {},
  "failed": false,
  "inited": true,
  "hexId": "0x0148-0x0004-0x0001",
  "dbLink": "https://devices.zwave-js.io/?jumpTo=0x0148:0x0004:0x0001:0.1",
  "manufacturerId": 328,
  "productId": 1,
  ```

**Comment by AlCalzone** [comment]
@zwave-js-bot config help wanted

**Comment by AlexCPU** [comment]
@AlCalzone this was fixed by #4099 and can be closed.
